### PR TITLE
Introduce keep_data instead of keep_all and keep_failed

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

**Comment by mstajdohar** [comment]
@jkokosar, @JenkoB, note the change.

**Comment by mstajdohar** [comment]
Closes #213.

**Comment by tjanez** [comment]
Looks good so far

File: resolwe/test/testcases.py [hunk 1]
```diff
@@ -146,29 +147,31 @@ def setUp(self):
...
             shutil.rmtree(settings.FLOW_EXECUTOR['DATA_DIR'], ignore_errors=True)
 
         self.settings.disable()
 
         super(TransactionTestCase, self).tearDown()
 
-    def keep_all(self):
-        """Do not delete output files after test for all data."""
-        self._keep_all = True
+    def keep_data(self, mock_purge=True):
```
Comment (dblenkus) [inline resolwe/test/testcases.py]: I don't like `mock_purge` name. User who will use this doesn't need to know what mock is. It should describe what it does, not how it does.

**Pull request merged**